src/platforms/common/data-management/sensitive-data/index.mdx

```diff
@@ -66,23 +66,15 @@ For more details, see [_Filtering Events_](../../configuration/filtering/).
 
 Instead of sending confidential information in plaintext, considering hashing it:
 
-```javascript
-Sentry.setTag("birthday", checksum_or_hash("08/12/1990"));
-```
+<PlatformContent includePath="sensitive-data/set-tag" />
 
 This will allow you to correlate it within internal systems if needed, but keep it confidential from Sentry.
 
 **User details**
 
 Your organization may determine that emails are not considered confidential, but if they are, consider instead sending your internal identifier:
 
-```javascript
-Sentry.setUser({ id: user.id });
-
-// or
-
-Sentry.setUser({ username: user.username });
-```
+<PlatformContent includePath="sensitive-data/set-user" />
```

Comment:
```suggestion
<PlatformContent includePath="set-user" />
```

Reply:
we could use the already defined `set-user` but it's not exactly the same example, wdyt?